bzt/requests_model.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, config, scenario, engine):
             file_dict["path"] = path
 
             mime = mimetypes.guess_type(file_dict["path"])[0] or "application/octet-stream"
-            file_dict.get('mime-type', mime)
+            file_dict["mime-type"] = mime
```

Comment:
This changes the behavior. Original behavior was "set if not set", while new is "just set".